src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-CSharp/wwwroot/css/site.css

```diff
@@ -7,9 +7,9 @@ a.navbar-brand {
   word-break: break-all;
 }
 
-/* Provide sufficient contrast against white background */
+/* Provide sufficient contrast against white background and black text */
```

Comment:
Personally I don't mind whether you change this or not but optional suggestion:

```suggestion
```

We don't explain the other CSS rules so it's odd to have a comment explaining this one.

Reply:
Do you want me to just remove it everywhere?

Reply:
I would, but really I'm fine with whatever you choose.

Reply:
I'll remove it, I thought it was odd too, especially since we only comment it in 3 of the places we set it